front/app/containers/Admin/projects/project/analysis/Insights/Summary.tsx

```diff
@@ -69,6 +76,24 @@ const Summary = ({ summary }: Props) => {
   };
 
   const hasFilters = !!Object.keys(summary.attributes.filters).length;
+  const tagId = summary.attributes.filters.tag_ids
+    ? summary.attributes.filters.tag_ids[0]
+    : null;
+
+  const tag = tags?.data.find((tag) => tag.id === tagId);
+
+  const phaseId = searchParams.get('phase_id');
+
+  const handleRestoreFilters = () => {
+    setSearchParams(
+      phaseId
+        ? {
+            phase_id: phaseId,
+          }
+        : {}
+    );
```

Comment:
I don't get why this is necessary?

Reply:
Because the phase id parameter is used by the go-back button. When we restore the filters, we are clearing the url but this part should be kept, if it's there.